### PR TITLE
chore(lighthouse): JIRA-9353 Use specific version

### PR DESCRIPTION
**Pull request by lukasjuhas**
Use a specific version over dev-master for lighthouse.

Relates to: https://worksome.atlassian.net/browse/JIRA-9246, so we can update on the platform.

### CODE REVIEW

**Comment by lukasjuhas** [comment]
Okay, I can see it's failing due to my change... not sure what's going on, thought 🤔 . Or did we want to specifically use the upcoming 6 version?

**Comment by owenvoke** [comment]
We are specifically using the upcoming version (due to new features that we have already been using in production), I'm working on changes for our other packages to use `^6.0.0`, as v6.0.0-beta.1 has been released so in theory not much will change before the proper release. 👍🏻 And in platform, we can then lock to a versioned tag.

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": "^8.2",
         "brick/math": "^0.9.3",
-        "nuwave/lighthouse": "dev-master"
+        "nuwave/lighthouse": "^5.71.0"
```
Comment (owenvoke) [inline composer.json:21]: It should be safe to use this:
```suggestion
        "nuwave/lighthouse": "^6.0.0"
```

We'd then need to release a new version, and update to `6.0.0-beta.1` in platform.

**Comment by lukasjuhas** [comment]
> We are specifically using the upcoming version (due to new features that we have already been using in production), I'm working on changes for our other packages to use `^6.0.0`, as v6.0.0-beta.1 has been released so in theory not much will change before the proper release. 👍🏻 And in platform, we can then lock to a versioned tag.
> 
> The reason it's failing on CI is because v6 adds stricter typing to properties such as `$description`.
> 
> For example, here are my PRs for updating Authsome:
> 
> * [chore(deps): update Lighthouse to 6.x pest-graphql-coverage#5](https://github.com/worksome/pest-graphql-coverage/pull/5)
> * [chore: update Lighthouse to v6.0.0-beta.1 authsome#614](https://github.com/worksome/authsome/pull/614)

I see; that makes a lot of sense! Thank you. Would you like me to update this PR or close it and let you deal with all the instances?

**Comment by owenvoke** [comment]
Up to you. I'm happy to take this on, but applying the change I mentioned should resolve this. So makes sense to just keep a single PR. 👍🏻

**Review by owenvoke — APPROVED** [review]

**Pull request closed without merging**